app/Console/Commands/SyncMedia.php

```diff
@@ -63,24 +63,26 @@ public function logToConsole($path, $result)
     {
         $name = basename($path);
 
-        if ($result === true) {
+        if (is_null($result)) {
```

Comment:
bacause you use if (...true) and elseif (false) another option, only null or diff bool